shell/platform/darwin/macos/framework/Source/FLEExternalTextureGL.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Foundation/Foundation.h>
+
+#import "flutter/shell/platform/darwin/macos/framework/Headers/FLETexture.h"
+#import "flutter/shell/platform/embedder/embedder.h"
+
+/**
+ * Used to bridge FLETexture object and handle the texture copy request.
```

Comment:
Please provide more context in this comment:
- Bridge between what and what?
- What texture copy request?

Providing some high-level explanation of how this class fits into the overall flow of passing textures back and forth wit the engine would be very helpful.